### PR TITLE
Issue 1673

### PR DESCRIPTION
**Pull request by sowmiyasubramaniyan**
Issue fix: #1673

### CODE REVIEW

**Review by SarfrazS — APPROVED** [review]
Before fix:-
t1

After fix:-
t2

File: resources/views/themes/default1/front/auth/login-register.blade.php [hunk 1]
```diff
@@ -1481,9 +1482,12 @@ function getCountryAttr(val) {
 
         function getState(val) {
             $.ajax({
+                headers: {
+                   'X-CSRF-TOKEN': $('meta[name="csrf-token"]').attr('content')
+                },
                 type: "GET",
                 url: "{{url('get-loginstate')}}/" + val,
-                data: {'country_id':val,'_token':"{{csrf_token()}}"},//'country_id=' + val,
+                data: {'country_id':val},//'country_id=' + val,
```
Comment (sandesh556) [inline resources/views/themes/default1/front/auth/login-register.blade.php:1490]: @sowmiyasubramaniyan you have to csrf token to the form. Not just remove it from the URL. Now this is vulnerable to cross-site attack

File: resources/views/themes/default1/front/auth/login-register.blade.php [hunk 2]
```diff
@@ -1497,9 +1501,12 @@ function getState(val) {
 
         function getCode(val) {
             $.ajax({
+                headers: {
+                   'X-CSRF-TOKEN': $('meta[name="csrf-token"]').attr('content')
+                },
                 type: "GET",
                 url: "{{url('get-code')}}",
-                data: {'country_id':val,'_token':"{{csrf_token()}}"},//'country_id=' + val,
+                data: {'country_id':val},//'country_id=' + val,
```
Comment (sandesh556) [inline resources/views/themes/default1/front/auth/login-register.blade.php:1509]: @sowmiyasubramaniyan you have to csrf token to the form. Not just remove it from the URL. Now this is vulnerable to cross-site attack

**Review by sandesh556 — CHANGES_REQUESTED** [review]

File: resources/views/themes/default1/front/auth/login-register.blade.php (hunk 2 above)
Comment (sowmiyasubramaniyan) [reply resources/views/themes/default1/front/auth/login-register.blade.php:1509]: image

**Review by sandesh556 — APPROVED** [review]

**Review by SarfrazS — APPROVED** [review]
working fine.

**Review by bindu95 — APPROVED** [review]
**BEFORE FIX:** We are sending token in the request URL
Screenshot 2022-05-16 at 4 11 15 PM


**AFTER FIX:** We are not sending tokens in the request URL
Screenshot 2022-05-16 at 4 11 50 PM

**Review by RafficMohammed — APPROVED** [review]

**Pull request merged**